tests/unit/info.tcl

```diff
@@ -369,5 +369,29 @@ start_server {tags {"info" "external:skip"}} {
             assert_equal [getInfoProperty $info client_output_buffer_limit_disconnections] {1}
             r config set client-output-buffer-limit $org_outbuf_limit
         } {OK} {logreqres:skip} ;# same as obuf-limits.tcl, skip logreqres
+
+        test {clients: pubsub clients} {
+            set info [r info clients]
+            assert_equal [getInfoProperty $info pubsub_clients] {0}
+            set rd1 [redis_deferring_client]
+            set rd2 [redis_deferring_client]
```

Comment:
we're missing the `$rd1 close` at the end.
and then maybe validate the pubsub_clients count goes back to 0 (with a wait_for_condition?)